src/diffusers/models/attention_processor.py

```diff
@@ -1762,7 +1762,9 @@ def __call__(
             query = apply_rotary_emb(query, image_rotary_emb)
             key = apply_rotary_emb(key, image_rotary_emb)
 
-        hidden_states = F.scaled_dot_product_attention(query, key, value, dropout_p=0.0, is_causal=False)
+        hidden_states = F.scaled_dot_product_attention(
```

Comment:
i see you added this to F.scaled_dot_product_attention
but just wonder how would this help? it is currently not passing down from the flux transformer to attenton_processor https://github.com/huggingface/diffusers/blob/c96bfa5c80eca798d555a79a491043c311d0f608/src/diffusers/models/attention.py#L190